examples/GainPlugin/jsui/src/Meter.js

```diff
@@ -1,11 +1,5 @@
-import React, { Component } from 'react';
-import {
-  EventBridge,
-  Image,
-  Text,
-  View,
-} from 'react-juce';
-
+import React, { Component } from "react";
+import { EventBridge, Image, Text, View } from "react-juce";
```

Comment:
This is super nit-picky of me, but I prefer this type of multiple-import to be separated by newlines so that if you change which things you import here it's very clearly a one-liner change in the git diff, versus having to parse out which part of the long line changed.

Can prettier adapt to that? I don't want to hold this up on account of that, but so far that's my only style preference

Reply:
I'm going to go ahead and merge, but if there's a flag we can flip for this one I may prefer that we do that in a follow-up pull request

Reply:
I know what you mean, just took a look at prettier config options and nothing jumped out at me.  It will break to multi-line if the list of imports is long enough.  In the single-line case, if the changes aren't massive, I think you'll get pretty meaningful highlighting out of github by default too: https://github.blog/2014-09-04-better-word-highlighting-in-diffs/